dbms/src/IO/WriteBufferFromOStream.cpp

```diff
@@ -19,14 +19,16 @@ void WriteBufferFromOStream::nextImpl()
     ostr->flush();
 
     if (!ostr->good())
-        throw Exception("Cannot write to ostream", ErrorCodes::CANNOT_WRITE_TO_OSTREAM);
+        throw Exception("Cannot write to ostream at " + std::to_string(offset()) + "/" + std::to_string(size),
```

Comment:
This is offset from the start of buffer; size is the size of buffer - not the size of file as you expected.
This error message will be completely useless and confusing.

Reply:
`123456/293` is non obvious.